cmd/helm/template.go

```diff
@@ -104,10 +114,19 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 
 							// if the filepath provided matches a manifest path in the
 							// chart, render that manifest
-							if f == manifestPath {
-								manifestsToRender = append(manifestsToRender, manifest)
-								missing = false
+							if strings.HasSuffix(f, "*") {
```

Comment:
It looks to me like you could remove the `strings.HasSuffix(...)` guard here, and just let all tests go through `filepath.Match`. Then you also wouldn't need your last`if f != manifestPath` check. This would have the added advantage of also allowing `?` and other match characters. Up to you though. I don't think this is blocking.

Reply:
Yes, you're right. I added this suffix check to allow specifying directories without glob, but this approach is way more versatile.